docs/src/production_readiness.md

```diff
@@ -0,0 +1,93 @@
+# Production Readiness Guide
+
+Running a production grade PostgreSQL system requires effort and preparation. If
+you're coming to CloudNativePG with PostgreSQL experience, or with other
+relational database  experience, you should already know much of what you need
+to have a production-ready CNPG system, with the additional benefits brought by
```

Comment:
I think we should either say CloudNativePG (CNPG) at the top and then just use CNPG in the doc or use CloudNativePG everywhere. Only commenting on this one but there are a few more usages of CNPG